### PR TITLE
fix: put wds theme provider under feature flag

### PR DESCRIPTION
**Pull request by KelvinOm**
## Description
Put WDS theme provider under the feature flag so that WDS don't affect prod anymore.

Also fix this 
CleanShot 2023-12-15 at 13 50 29@2x

**How it look like now**

WDS enabled
Снимок экрана 2023-12-15 в 14 34 54
Снимок экрана 2023-12-15 в 14 34 59

WDS disabled
Снимок экрана 2023-12-15 в 14 35 59
Снимок экрана 2023-12-15 в 14 36 03

## Summary by CodeRabbit

- **Refactor**
  - Improved feature flag naming for clarity in the app viewer settings.
  - Simplified theme background color application in the editor canvas.
  - Centralized rendering logic for editor canvas components for better maintainability.

- **Style**
  - Streamlined the application of themes to ensure consistent background colors across the platform.

### CODE REVIEW

File: app/client/src/pages/Editor/Canvas.tsx [hunk 1]
```diff
@@ -90,31 +82,40 @@ const Canvas = (props: CanvasProps) => {
...
-              classList:
-                layoutSystemType === LayoutSystemTypes.ANVIL
-                  ? ["main-anvil-canvas"]
-                  : [],
-            } as WidgetProps)}
-        </Wrapper>
-      </WDSThemeProvider>
+      <Wrapper
+        $enableMainCanvasResizer={!!props.enableMainCanvasResizer}
+        background={isWDSEnabled ? "" : backgroundForCanvas}
```
Comment (jsartisan) [inline app/client/src/pages/Editor/Canvas.tsx:90]: shouldn't this be `var(--color-bg)` when isWDSEnabled?

Comment (KelvinOm) [reply app/client/src/pages/Editor/Canvas.tsx:90]: Nope, provider has this token by default.

Comment (jsartisan) [reply app/client/src/pages/Editor/Canvas.tsx:90]: Ok got it.

**Review by jsartisan — DISMISSED** [review]

**Review by riodeuno — APPROVED** [review]
Looks good to me.

@KelvinOm I was curious, about why we chose to make `renderChildren` a function, instead of using a constant?

**Comment by KelvinOm** [comment]
> Looks good to me.
> 
> @KelvinOm I was curious, about why we chose to make `renderChildren` a function, instead of using a constant?

@riodeuno As far as I know, there are no special benefits from using one method or another. We've been using functions in WDS lately, so I just did it by analogy.

**Review by riodeuno — APPROVED** [review]

**Pull request merged**